Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-list/SubmissionsList.tsx

```diff
@@ -42,7 +42,7 @@ const SubmissionsList = ({
         [ submissionListItemClass ],
     );
     const submissionBtnClass = 'submissionBtn';
-    const submissionsLabelTypeClassName = 'submissionTypeLabel';
+    // const submissionsLabelTypeClassName = 'submissionTypeLabel';
```

Comment:
For tests?

Reply:
its unused in the rendering of the submission list